### DISCUSSION TITLE
Car shop

### DISCUSSION THREAD

**Discussion by mmdkvz** [OwlGamingCommunity/MTA#12] [Support]
i have installed a mta owl gaming
how vehicle shop work ??
Please help me 😭

**Comment by saviorsoldier** [comment]
You need to add vehicles to shops via Vehicle Library. Also, don't forget to check carshop-system & vehicle_manager under [vehicle] file.

**Reply by mmdkvz** [reply]
I am added a vehicle in vehicle_shop in sql
But cars not enabled in the game
/vehlib and enabled = no

**Reply by saviorsoldier** [reply]
Then you need to make it enabled. Everything you need is in /vehlib. You don’t need to mess up with SQL to add a vehicle.

**Reply by mmdkvz** [reply]
> Then you need to make it enabled. Everything you need is in /vehlib. You don’t need to mess up with SQL to add a vehicle.

I did it with this command but it says it is not enable

**Comment by wlittlefield95** [comment]
where do i fix this in the carshop-system ?

**Reply by wlittlefield95** [reply]
<img width="1920" height="1080" alt="mta-screen_2025-07-12_04-03-54" src="https://github.com/user-attachments/assets/f5db4ce9-5bd8-47ce-9640-e87024e1f50d" />
